server/lib/tuist/github_app_installations.ex

```diff
@@ -0,0 +1,62 @@
+defmodule Tuist.GitHubAppInstallations do
```

Comment:
I lean on having this one under a `VCS` context/module, representing the interface of Tuist with version control systems.

Reply:
sure 👍